cups/dest.c

```diff
@@ -3400,21 +3413,63 @@ cups_dnssd_unquote(char       *dst,	/* I - Destination buffer */
 }
 
 
+/*
+ * 'cups_gettimeofday()' - Get the sharpest time possible
+ */
+
+static void
+cups_gettimeofday(_cups_time_t *t)
+{
+#if _WIN32
+  t->t = GetTickCount();
+#elif HAVE_CLOCK_MONOTONIC
+  clock_gettime(CLOCK_MONOTONIC, &t->t);
```

Comment:
Even if `CLOCK_MONOTONIC` is defined, that doesn't mean that the platform will support that clock. Check for a non-zero return and fallback on gettimeofday when it fails.